### PR TITLE
Fix the PostgreSQL 12 package name

### PR DESCRIPTION
**Pull request by spetrosi**
Bug 1898111 - Fix typo in upgrade guide for rh-postgresql12-postgresql-evr

https://bugzilla.redhat.com/show_bug.cgi?id=1898111

### CODE REVIEW

File: guides/common/modules/proc_installing-postgresql.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ include::snip_firewalld.adoc[]
 ----
 # yum install rh-postgresql12-postgresql-server \
 rh-postgresql12-syspaths \
-rh-postgresql12-postgresql-evr
+rh-postgresql12-postgresql-server
```
Comment (ekohl) [inline guides/common/modules/proc_installing-postgresql.adoc:15]: This doesn't make sense to me. This package is already listed on line 13 and the evr extension is really needed. On an all-in-one installation the installer can take care of it, but on an external server you need to do it manually.

**Review by ekohl — CHANGES_REQUESTED** [review]
I think the Bugzilla is invalid.

File: guides/common/modules/proc_installing-postgresql.adoc (hunk 1 above)
Comment (spetrosi) [reply guides/common/modules/proc_installing-postgresql.adoc:15]: This contradicts what Kenny says in [BZ#1898111](https://bugzilla.redhat.com/show_bug.cgi?id=1898111)

Comment (ekohl) [reply guides/common/modules/proc_installing-postgresql.adoc:15]: And I think that BZ is invalid. I've left a comment there.

Comment (pipopopo) [reply guides/common/modules/proc_installing-postgresql.adoc:15]: My mistake seems that evr contains the server as a dependency.

# rpm -qpR rh-postgresql12-postgresql-evr-0.0.2-1.el7sat.x86_64.rpm 
~~~
rh-postgresql12-postgresql-server
rh-postgresql12-runtime
rh-postgresql12-runtime
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
~~~

**Comment by spetrosi** [comment]
The bug has been closed as NOTABUG

**Pull request closed without merging**